packages/node-core/src/indexer/storeCache/storeModel.service.ts

```diff
@@ -19,19 +20,29 @@ export interface IStoreModelService {
   poi: IPoi | null;
   metadata: IMetadata;
 
+  init(historical: boolean, useCockroachDb: boolean, meta: MetadataRepo, poi?: PoiRepo): void;
+
   getModel<T extends BaseEntity>(entity: string): IModel<T>;
 
   // addExporter(entity: string, exporterStore: CsvStoreService): void;
 
   applyPendingChanges(height: number, dataSourcesCompleted: boolean): Promise<void>;
 
   updateModels({modifiedModels, removedModels}: {modifiedModels: ModelStatic<any>[]; removedModels: string[]}): void;
+
+  resetData?(): Promise<void>;
+
+  flushData?(forceFlush?: boolean): Promise<void>;
```

Comment:
Optional methods shouldn't exist on the interface. If methods are needed then determine the implementation instead